docs/guides/mobile-plugin-ima-android.md

```diff
@@ -29,21 +29,25 @@ This plugin is in the following open-source GitHub repository:
 
 To run this plugin, install the following:
 
-- [Google IMA SDK v3.29.0](https://developers.google.com/interactive-media-ads/docs/sdks/android/client-side/history) (ONLY 3.29.0: not any higher version)
-- [UID2 Android SDK v0.4.0](https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk-ima)  (for documentation, see [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md))
-- UID2 IMA Plugin for Android v0.1.0
+- [Google IMA SDK v3.30.3](https://developers.google.com/interactive-media-ads/docs/sdks/android/client-side/history)
+- [UID2 SDK for Android v0.5.0](https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk) (for documentation, see [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md))
+- [UID2 IMA Plugin for Android v0.5.0](https://central.sonatype.com/artifact/com.uid2/uid2-android-sdk-ima)
 
 ## Installation
 
 Prerequisite: Install the Google Interactive Media Ads SDK and the UID2 Android SDK.
 
-Install the UID2 Android IMA Plugin v0.1.0 via Gradle / Maven to an existing app with the UID2 Android SDK and Google IMA SDK installed.
+Install the UID2 Android IMA Plugin v0.4.0 to an existing app with the UID2 Android SDK and Google IMA SDK installed. There are two installation options:
```

Comment:
The linked IMA plugin is 0.5.0, maybe update?

Reply:
Fixed thx. No need to mention the version here -- I removed it.